assets/src/data/eventespresso/core/resolvers/relations.js

```diff
@@ -87,9 +87,26 @@ export function* getRelatedEntities( entity, relationModelName ) {
 		'receiveRelationEndpointForModelEntity',
 		[ modelName, entity.id, pluralRelationName, relationEndpoint ]
 	);
-	const relationEntities = yield fetch( {
+	let relationResponseType = yield resolveSelect(
+		SCHEMA_REDUCER_KEY,
+		'getRelationResponseType',
+		modelName,
+		relationModelName,
+	);
+
+	relationResponseType = relationResponseType === '' ?
+		'array' :
+		relationResponseType;
+
+	let relationEntities = yield fetch( {
 		path: relationEndpoint,
 	} );
+
+	relationEntities = relationResponseType === 'object' &&
+		! isEmpty( relationEntities ) ?
+		[ relationEntities ] :
+		relationEntities;
```

Comment:
What else can be returned when fetching `relationEntities`?
Can `null` be returned?
Because if so, then it would remain as `null` instead of being `[ null ]`...
which might be exactly what you want, but I'm just checking to be sure.

Would it possibly be better to simply check whether or not `relationEntities` is an `array`, and if not, then put it in one?
ie:

```js
relationEntities = relationResponseType === 'array' &&
		! Array.isArray( relationEntities ) ?
		[ relationEntities ] :
		relationEntities;
```

or even just:

```js
relationEntities = ! Array.isArray( relationEntities ) ?
		[ relationEntities ] :
		relationEntities;
```

Reply:
Ya this needs improved a bit.  Going to rework.

Reply:
ya that looks pretty bullet proof to me now